### PR TITLE
wrapFunctionCalls is enabled postOpt

### PR DESCRIPTION
**Pull request by jhag**
This module introduces variables for each function call and substitutes all these calls with the newly introduced variables. Some special cases are skipped and variables are not introduced.

### CODE REVIEW

**Comment by sjoelund** [comment]
You should squash/fixup your commits I guess. Or do you only want to see if your changes pass the tests?

**Comment by lochel** [comment]
Yes, this should get squashed to a single commit before pushing it. However, we can first run Hudson to check the testsuite.

File: Compiler/FrontEnd/Expression.mo [hunk 1]
```diff
@@ -9766,12 +9766,12 @@ algorithm
 end isExpCrefOrIfExp;
 
 public function isExpIfExp
-"Returns true if expression is an if expression"
+"Returns true if expression is an if or a smooth expression"
```
Comment (lochel) [inline Compiler/FrontEnd/Expression.mo]: Is that correct?

Comment (jhag) [inline Compiler/FrontEnd/Expression.mo]: Sorry, this line is wrong! I remove it when I fix it tomorrow!

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2016-04-28_10-46-58](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/2567/console).

**Comment by jhag** [comment]
We push it under reserve because we want to see the effects of the coverage test.

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2016-04-28_12-03-54](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/2570/console).

**Pull request closed without merging**